### ISSUE TITLE
Common IO Error IndyError, when creating configPool

### ISSUE THREAD

**Issue by bootstrapsp** [bootstrapsp/miu#11]
**Describe the bug**
When attempting to create config pool from gRPC client, Miu throws error, see below
**To Reproduce**
1. Start Miu pythong grpc_server/server.py
2. Provide require input for the gRPC client to be able to create pool, here's e.g. from go based gRPC client

> func CreatePoolLedgerConfig(conn *grpc.ClientConn) {
	cplc := pb.NewPoolServiceClient(conn)

	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)

	defer cancel()

	createPoolLedgerConf, err := cplc.CreatePoolLedgerConfig(ctx, &pb.CreatePoolLedgerConfigRequest{
		ConfigName: "ScuridPool1",
		Config:     &pb.ConfigCreatePoolLedger{
			GensisTxn: genTx,
		},
	})
3. Run gRPC client to connect to Miu on :5001 port

> DEBUG:indy.pool:create_pool_ledger_config: >>> config_name: 'ScuridPool1', config: '{"genesis_txn": REDACTED
DEBUG:indy.libindy:do_call: >>> name: indy_create_pool_ledger_config, args: (c_char_p(139674895488016), c_char_p(139674752459008), <CFunctionType object at 0x7f089858c5c0>)
DEBUG:indy.libindy:do_call: Function indy_create_pool_ledger_config returned err: 0
DEBUG:indy.libindy:do_call: <<< <Future pending>
INFO:indy.libindy.native.indy.commands:	src/commands/mod.rs:133 | PoolCommand command received
INFO:indy.libindy.native.pool_command_executor:	src/commands/pool.rs:62 | Create command received
DEBUG:indy.libindy.native.indy.commands.pool:	src/commands/pool.rs:138 | create >>> name: "ScuridPool1", config: Some(PoolConfig { genesis_txn: "REDACTED)
DEBUG:indy.libindy:_indy_callback: >>> command_handle: 4, err 114, args: ()
DEBUG:indy.libindy:_indy_callback: <<<
DEBUG:indy.libindy:_indy_loop_callback: >>> command_handle: 4, err 114, args: ()
WARNING:indy.libindy:_indy_loop_callback: Function returned error 114
DEBUG:indy.libindy:_indy_loop_callback <<<
ERROR:pool_service:Exception Occurred @ CreatePoolLedgerConfig ------
ERROR:pool_service:ErrorCode.CommonIOError
ERROR:root:Exception calling application: No positional arguments allowed
Traceback (most recent call last):
  File "/home/scurid/goingTheRightSide/miu/pyserverforindy/grpc_server/pool_service.py", line 67, in CreatePoolLedgerConfig
    resp = await indy_pool.create_pool_ledger_config(config_name, config)
  File "/home/scurid/goingTheRightSide/miu/pyserverforindy/scurid/lib/python3.5/site-packages/indy/pool.py", line 39, in create_pool_ledger_config
    create_pool_ledger_config.cb)
  File "/usr/lib/python3.5/asyncio/futures.py", line 361, in __iter__
    yield self  # This tells Task to wait for completion.
  File "/usr/lib/python3.5/asyncio/tasks.py", line 296, in _wakeup
    future.result()
  File "/usr/lib/python3.5/asyncio/futures.py", line 274, in result
    raise self._exception
indy.error.IndyError: ErrorCode.CommonIOError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "grpc_server/server.py", line 97, in _call_behavior
    return await behavior(argument, context), True
  File "/home/scurid/goingTheRightSide/miu/pyserverforindy/grpc_server/pool_service.py", line 72, in CreatePoolLedgerConfig
    return identitylayer_pb2.CreatePoolLedgerConfigResponse(resp)
TypeError: No positional arguments allowed

**Expected behavior**
Should create pool config

**Screenshots**
-NA-

**Desktop (please complete the following information):**
 - OS: ubuntu 16

**Smartphone (please complete the following information):**
-NA-

**Additional context**
-NA-

**Comment by vlmold** [comment]
> During handling of the above exception, another exception occurred:

this is fixed with #17 
```        
except IndyError as e:
            logger.error("Indy Exception Occurred @ CreatePoolLedgerConfig ------")
            logger.error(e.message) 
            return identitylayer_pb2.CreatePoolLedgerConfigResponse(ErrorCode=e.error_code )
```
Probably pool has already been created or genTx is not a valid path to genesis file.

**Comment by bootstrapsp** [comment]
thanks @vladyslavmunin hinting that was the case!
